etc/kayobe/environments/ci-multinode/inventory/group_vars/storage/network-interfaces

```diff
@@ -2,7 +2,7 @@
 ###############################################################################
 # Network interface definitions for the storage group.
 
-admin_oc_interface: "eth0"
+admin_oc_interface: "ens3"
```

Comment:
Greg already made this change - where did it go?

Reply:
I think the problem here is the branch I have been working hasn't kept in sync with the `multinode-env` so changes like this have been repeated.